apps/comps/components/Navbar/index.tsx

```diff
@@ -14,7 +14,7 @@ export const Navbar: React.FunctionComponent<{ children: React.ReactNode }> = ({
 
   const navItems = [
     { label: "COMPETITIONS", href: "/competitions" },
-    { label: "LEADERBOARDS", href: "/leaderboards" },
+    { label: "LEADERBOARDS", href: "/leaderboard" },
```

Comment:
@ramirotw i made a small tweak in https://github.com/recallnet/js-recall/pull/328 to fix the route since it was pointing to a non-existent page. i renamed the page from `leaderboard` to `leaderboards`. So, reverting this results in a 404 when you click the nav.

However, perhaps we should rename the backend `api` app to `/leaderboards` since it currently is `/leaderboard`.

Reply:
got it, let me change the link and also the API endpoint